lib/smart_app_launch/endpoints/mock_smart_server.rb

```diff
@@ -1,59 +1,54 @@
 require 'jwt'
 require 'faraday'
 require 'time'
+require 'base64'
+require 'rack/utils'
 require_relative '../urls'
 require_relative '../tags'
+require_relative '../client_suite/client_options'
 
 module SMARTAppLaunch
   module MockSMARTServer
-    SUPPORTED_SCOPES = ['openid', 'system/*.read', 'user/*.read', 'patient/*.read'].freeze
+    SUPPORTED_SCOPES = ['system/*.read', 'user/*.read', 'patient/*.read'].freeze
 
     module_function
 
     def smart_server_metadata(suite_id)
       base_url = "#{Inferno::Application['base_url']}/custom/#{suite_id}"
       response_body = {
         token_endpoint_auth_signing_alg_values_supported: ['RS384', 'ES384'],
-        capabilities: ['client-confidential-asymmetric'],
+        capabilities: ['client-confidential-asymmetric', 'launch-ehr' ,'launch-standalone', 'authorize-post',
+                       'client-public', 'client-confidential-symmetric', 'permission-offline', 'permission-online',
+                       'permission-patient', 'permission-user', 'permission-v1', 'permission-v2',
+                       'context-ehr-patient', 'context-ehr-encounter', 
+                       'context-standalone-patient', 'context-standalone-encounter',
+                       'context-banner', 'context-style'],
         code_challenge_methods_supported: ['S256'],
-        token_endpoint_auth_methods_supported: ['private_key_jwt'],
+        token_endpoint_auth_methods_supported: ['private_key_jwt', 'client_secret_basic'],
```

Comment:
What about `client_secret_post`?

Reply:
added in [742af51](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/96/commits/742af51aa15e06b30cc094a010658d3b287e036e)